solidity/dashboard/src/hooks/useWalletAddressFromUrl.js

```diff
@@ -0,0 +1,19 @@
+import { useLocation } from "react-router-dom"
+import web3Utils from "web3-utils"
+
+const useWalletAddressFromUrl = () => {
+  const location = useLocation()
```

Comment:
I think we can use [`useParams`](https://reactrouter.com/web/example/url-params) from the `react-router-dom` pacakge. eg:

```
const { address } = useParams() 
```

Reply:
Hmm I think we cannot because when we use the `useWalletAddressFromUrl` hook inside `useSubscribeToConnectorEvents`, which in turn is used in `AppLayout`, we are outside of a `<Routing/>` component which resulting in returning an empty object from `useParams()` 🤔

Reply:
I see. How about `useRouteMatch` ? It can be used outside the `Route` component and parses the url to params based on the provided path. Please see https://reactrouter.com/web/api/Hooks/useroutematch